src/CachePlugin.php

```diff
@@ -138,6 +137,40 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * Calculate the timestamp when this cache item should be dropped from the cache. The lowest value return after
+     * the calculation is $maxAge.
+     *
+     * @param int|null $maxAge
+     *
+     * @return int|null Unix system time. @see PSR6 for caching
```

Comment:
should we mention that null means the cache item never expires and is only evicted if the cache drops items before they expire?